### ISSUE TITLE
What is the standard way to get exact isActive() match in 1.0.0?

### ISSUE THREAD

**Issue by jedwards1211** [rackt/react-router#1561]
Here are my routes:

```
var routes = <Router history={history}>
  <Route component={App}>
    <Route path="/" component={Home}/>
    <Route path="/alerts" component={Alerts}/>
  </Route>
</Router>;
```

In my navbar, I have some Bootstrap-styled links:

```
<li className={this.isActive('/') ? 'active': ''}><Link to="/">Home</Link></li>
<li className={this.isActive('/alerts') ? 'active': ''}><Link to="/alerts">Alerts</Link></li>
```

As you might have guessed, when the user is at /alerts both links are highlighted as active.

What would be the standard way in 1.0.0 to determine if just `/` is active (and not any child route)?

**Comment by opatut** [comment]
I too am looking for this function, sadly, it seems not to be there yet. I guess someone has to start working [here](https://github.com/rackt/react-router/blob/master/modules/RouterContextMixin.js#L11) :wink:

**Comment by gonsfx** [comment]
+1

Should take route hierarchy into account, e.g.

```
  <Router history={new BrowserHistory}>
    <Route component={App}>
      <Route path="/" component={HomePage}>
        <Route path="nested" component={NestedPage}>
          <Route path="nested" component={NestedPage} />
        </Route>
        <Route path="nest" component={NestPage} />
      </Route>
      <Route path="some" component={SomePage} />
      <Route path="other" component={OtherPage} />
    </Route>
  </Router>
```

**Comment by knowbody** [comment]
let's continue the discussion in #1441, it will be great to get the PR with failing tests so it's easier to target the issues, but this one should be solved by #1842
